man/org.freedesktop.machine1.xml

```diff
@@ -394,7 +396,8 @@ node /org/freedesktop/machine1 {
 
       <para><function>CopyFromMachine()</function> copies files or directories from a container into the
       host. It takes a container name, a source directory in the container and a destination directory on the
-      host as arguments. <function>CopyToMachine()</function> does the opposite and copies files from a source
+      host as arguments. Optionally takes "--force" to force the copy if the destination already exists.
```

Comment:
well, the dbus iface doesn't take `--force` really...

Reply:
- [ ] Need to fix the documentation